Backend/BikesRentalServer.WebApi/Controllers/MalfunctionsController.cs

```
@@ -0,0 +1,66 @@
+        {
+            var response = _techsService.RemoveMalfunction(id);
+            return response.Status switch
+            {
+                Status.Success => Ok(new GetMalfunctionResponse
+                {
+                    Id = response.Object.Id.ToString(),
+                    Bike = response.Object.Bike is null ? null : new GetBikeResponse()
+                    {
+                        Id = response.Object.Bike.Id.ToString(),
+                        Station = response.Object.Bike.Station is null ? null : new GetStationResponse()
+                        {
+                            ActiveBikesCount = response.Object.Bike.Station.Bikes.Count,
+                            Id = response.Object.Bike.Station.Id.ToString(),
+                            Name = response.Object.Bike.Station.Name,
+                            Status = response.Object.Bike.Station.Status,
+                        },
+                        Status = response.Object.Bike.Status,
+                        User = response.Object.Bike.User is null ? null : new GetUserResponse()
+                        {
+                            Id = response.Object.Bike.User.Id.ToString(),
+                            Name = response.Object.Bike.User.Username,
+                        }
+                    },
+                    Description = response.Object.Description,
+                    DetectionDate = response.Object.DetectionDate,
+                    MalfunctionState = response.Object.State,
+                    ReportingUser = response.Object.ReportingUser is null ? null : new GetUserResponse()
+                    {
+                        Id = response.Object.ReportingUser.Id.ToString(),
+                        Name = response.Object.ReportingUser.Username,
+                    }
+                }),
+                Status.EntityNotFound => NotFound(response.Message),
+                Status.InvalidState => UnprocessableEntity(response.Message),
```

`RemoveMalfunction` does not return `Status.InvalidState` - it should then throw `InvalidOperationException` alongside `Status.UserBlocked` and invalid values